uint32 num_frames
float64 timeout
---
string success
uint16 num_frames
float64 width